build_for_sdk.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+xcodebuild ARCHS="armv7 armv7s arm64" ONLY_ACTIVE_ARCH=NO -configuration Release -project sqlcipher.xcodeproj
+xcodebuild ARCHS="x86_64 i386" ONLY_ACTIVE_ARCH=NO -configuration Release -project sqlcipher.xcodeproj -sdk iphonesimulator
+lipo -create build/Release-iphoneos/libsqlcipher.a build/Release-iphonesimulator/libsqlcipher.a -output libsqlcipher.a
```

Comment:
Cool - but I don't expect us to run that 3 lines script more than once a year so we can probably live with the hard-coded values.